code/modules/mob/living/carbon/xenomorph/attack_alien.dm

```diff
@@ -205,6 +219,14 @@
 	return TRUE
 
 
+/mob/living/carbon/Xenomorph/proc/process_rage_attack()
+	if(!istype(src,/mob/living/carbon/Xenomorph/Ravager))
```

Comment:
`return FALSE` and redefine for `/mob/living/carbon/Xenomorph/Ravager/process_rage_attack()`